cf-monitord/history.c

```diff
@@ -74,32 +74,34 @@ static void PutRecordForTime(CF_DB *db, time_t time, const Averages *values)
     WriteDB(db, timekey, values, sizeof(Averages));
 }
 
-static void Nova_SaveFilePosition(char *name, long fileptr)
+static void Nova_SaveFilePosition(const char *handle, char *name, long fileptr)
```

Comment:
`name` should also be `const`, FYI. If it was correct before, it would be easier for you.

Reply:
OK ill just fix it while I am here then. The fileptr should not be const?